habitat/config/default.py

```diff
@@ -93,11 +97,11 @@ def __init__(self, *args, **kwargs):
 # -----------------------------------------------------------------------------
 # # COMPOSITE TASK CONFIG
 # -----------------------------------------------------------------------------
-_C.TASK.TASK_SPEC_BASE_PATH = "configs/tasks/rearrange/pddl/"
+_C.TASK.TASK_SPEC_BASE_PATH = f"{_HABITAT_CFG_DIR}/tasks/rearrange/pddl/"
```

Comment:
Not sure that is good in longterm. It will forge absolute private path into the config and once dumped it will be hard to reuse for others. In these terms relative paths are better. Maybe leave it as it was.